src/pages/CategoriesPage/DeleteCategory/DeleteCategoryComponent.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Grid,
+} from '@mui/material';
+import { styles } from '../styles';
+
+interface Props {
+  deleteModalOpen: boolean;
+  onToggleDeleteStatusClick: () => void;
+  onSubmit: () => void;
+}
+
+const DeleteCategoryComponent = (props: Props) => {
+  return (
+    <Dialog open={props.deleteModalOpen}>
+      <div style = {styles.dialogTitleWrapper}>
+        <DialogTitle style={styles.dialogTitle}>
+          Click on the confirmation button to delete the category
+        </DialogTitle>
+      </div>
+      <DialogContent>
+        <Grid container>
+          <Grid item xs={6}>
+            <div style={styles.buttonsWrapper}>
```

Comment:
Do not use DIV. Use Box instead.

Reply:
All divs in the modal windows have been replaced by the Box components